File: Java/Q20.java
class Main {
    public static void main(String[] args) {
        int drink_choice = 3;
        switch (drink_choice) {
            case 1: System.out.println("You have selected: Tea"); break;
            case 2: System.out.println("You have selected: Coffee"); break;
            case 3: System.out.println("You have selected: Juice"); break;
            default: System.out.println("Invalid selection.");
        }
    }
}

File: CPP/Q20.cpp
#include <iostream>

int main() {
    int drink_choice = 3;
    switch (drink_choice) {
        case 1: std::cout << "You have selected: Tea" << std::endl; break;
        case 2: std::cout << "You have selected: Coffee" << std::endl; break;
        case 3: std::cout << "You have selected: Juice" << std::endl; break;
        default: std::cout << "Invalid selection." << std::endl;
    }
    return 0;
}